geoapi/tests/external_data_tests/test_external_data.py

```diff
@@ -449,16 +449,16 @@ def test_is_member_of_rapp_project_folder():
     assert not is_member_of_rapp_project_folder("/something/test.jpg")
 
 
-def test_get_additional_files_none(agave_utils_with_geojson_file):
-    assert not get_additional_files("testSystem", "/testPath/file.jpg", agave_utils_with_geojson_file)
+def test_get_additional_files_none(shapefile_fixture, agave_utils_with_geojson_file):
+    assert not get_additional_files(shapefile_fixture, "testSystem", "/testPath/file.jpg", agave_utils_with_geojson_file)
 
 
-def test_get_additional_files(agave_utils_with_geojson_file):
-    files = get_additional_files("testSystem", "/testPath/file.shp", agave_utils_with_geojson_file)
+def test_get_additional_shapefiles_files(shapefile_fixture, agave_utils_with_geojson_file):
```

Comment:
If 461 and 494 are correct, maybe line 456 needs to be `test_get_additional_files_shapefiles`?

Reply:
Feels like this is the correct change.